docs/translation/index.md

```diff
@@ -205,6 +219,10 @@ Typography is different in different languages and thus must be translated along
     {ts}Event Total:{/ts}
     ```
 
+### Unless unusably broken, consider leaving existing bad uses of ts unchanged
```

Comment:
It was related in that it came out of the same original discussion and the `<a>` links seemed to be more susceptible to it, but I can move it to a different PR.

Reply:
I'm not going to "stand in the way" of the PR because it's in there. If it's been discussed elsewhere  let's leave it there - I'm just generally nervous about telling people "don't fix that broken thing, it's too much work" which really isn't a good message to send.

Reply:
Maybe I can try to clarify in the text what "broken" means in this context and not use that word. That word is overused/misused in IT all the time, and obviously I do it too...

Reply:
Worth a try - but as I say I'll commit as is so I'll leave it up to you :-)